### ISSUE TITLE
Missing rtmidi package in  install_requirements

### ISSUE THREAD

**Issue by noahfx** [olemb/mido#96]
Hi,
Currently I am the maintainer of mido in Debian, I was building the latest version (1.2.6) and I have noticed the tests are failing due to a rtimidi import in ```mido/backends/rtmidi.py```: 

```
$ python3 -m pytest 
========================================================= test session starts ==========================================================
platform linux -- Python 3.5.3, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /home/noahfx/debian/python-mido/python-mido, inifile: setup.cfg
collected 81 items / 1 errors 

 ERROR collecting mido/backends/test_rtmidi.py _____________________________________________
ImportError while importing test module '/home/noahfx/debian/python-mido/python-mido/mido/backends/test_rtmidi.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
mido/backends/test_rtmidi.py:1: in <module>
    from .rtmidi import _expand_alsa_port_name
mido/backends/rtmidi.py:8: in <module>
    import rtmidi
E   ImportError: No module named 'rtmidi'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

``` 

I have made a PR that fixes this issue https://github.com/olemb/mido/pull/97 

Regards

**Comment by olemb** [comment]
Thank you for creating a Debian package!

Ah, I didn't consider that the test would require the `python-rtmidi` package. I've moved the function that's tested into a separate module to remove the dependency and will release a new version very soon (hopefully today).
 
The reason why `python-rtmidi` is not a dependency is that it's common to use Mido with another backend or without a backend at all.

I've been thinking of adding it as an extra by adding something like this:

```
    extras_require={
        'rtmidi':  ["python-rtmidi", "rtmidi"],
    }
```

(No sure if that's the exact syntax.)

**Comment by olemb** [comment]
It makes sense though for the Debian package to require RtMidi since the Debian RtMidi package will always be installable.

**Comment by olemb** [comment]
I've released 1.2.7 so I'm closing this and the pull request.
